### DISCUSSION TITLE
https://korhonen.cc/pgp/

### DISCUSSION THREAD

**Discussion by giscus** [FunctionalHacker/korhonen.cc#1] [Announcements]
# https://korhonen.cc/pgp/

I like PGP and I secure everything I can with it. If you would like to send me encrypted files, emails or anything of that sort, you came to the right page. Next up is my public key in different formats.
Fingerprint 1EFC4DBB079407113A761DF7C514037F1EFF4FA6
This is my key fingerprint and it can be found around the various keyservers on the internet but since the demise of SKS keyservers, I have not really bothered with keyservers so see below for the full key.
ASCII Armor Click here
OpenKeychain The following QR code should be importable in the Android app OpenKeyChain but this relys on keyservers so your results may vary.

https://korhonen.cc/pgp/

**Comment by FunctionalHacker** [comment]
Just testing the commenting feature :)
